settings.gradle.kts

```diff
@@ -1,5 +1,9 @@
 rootProject.name = "sigstore-java-root"
 
+if (JavaVersion.current() < JavaVersion.VERSION_17) {
+    throw UnsupportedOperationException("Please use Java 17 or 21 for launching Gradle when building sigstore-java, the current Java is ${JavaVersion.current().majorVersion}")
```

Comment:
We still will generate a java11 bytecode final result though? I'm not sure we can eliminate java 11 support just yet.

Reply:
Sure we generate Java 11-compatible bytecode (check for `targetJavaVersion`)

Reply:
See `release.set(buildParameters.targetJavaVersion)`